reap.sh

```diff
@@ -19,34 +19,41 @@ echo "  NS_BLACKLIST: ${NS_BLACKLIST[@]}"
 
 # get Jobs that do not have any parent resources (e.g. ignore those managed by CronJobs)
 IFS=$'\n'
-for j in $(kubectl get jobs --all-namespaces -o json | jq -r ".items[] | select( .metadata | has(\"ownerReferences\") | not) | [.metadata.name,.metadata.namespace,.metadata.creationTimestamp,.status.completionTime,.metadata.annotations.ttl] | @csv" | sed 's/"//g'); do
+for j in $(kubectl get jobs --all-namespaces -o json | jq -r ".items[] | [.metadata.name,.metadata.namespace,.metadata.creationTimestamp,.status.completionTime,.metadata.annotations.ttl,.status.active] | @csv" | sed 's/"//g'); do
```

Comment:
This PR looks generally really good! Quick question as to why removal of the `ownerReferences` check? I put that in to ignore Jobs that are managed by other resources (mainly CronJobs) as they have their own retention policy features anyway. Though I'm not opposed to having an env variable to control whether to ignore them.

Reply:
What if you want to remove the pods and jobs which this cronjob will be creating? The ownerReferences check will prevent deleting such jobs and many jobs will get piled up.

Reply:
@aramse Oh yes, you mean there is a configuration available for deleting jobs and pods created by cronjobs.
```
spec:
  successfulJobsHistoryLimit 10
  failedJobsHistoryLimit 0
```
I can re-add it. I didn't understand it at first.

Reply:
I just re-added the owner references. Please have a look. I have changed the logic flow too. That is to accomodate active and succeded change

Reply:
Thanks! I'll look to test and merge this in the next couple weeks, sorry been busy.